external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaContinuousStream.scala

```diff
@@ -53,7 +53,7 @@ class KafkaContinuousStream(
     failOnDataLoss: Boolean)
   extends ContinuousStream with Logging {
 
-  private val pollTimeoutMs = sourceOptions.getOrElse("kafkaConsumer.pollTimeoutMs", "512").toLong
+  private val pollTimeoutMs = sourceOptions.getOrElse(CONSUMER_POLL_TIMEOUT.key, "512").toLong
```

Comment:
These options that don't begin with "spark." and aren't from the Spark config shouldn't use the Spark config mechanism. Same for others below.